src/js/components/DateInput/DateInput.js

```diff
@@ -303,7 +301,11 @@ Use the icon prop instead.`,
         disabled={disabled}
         plain
         icon={icon || MaskedInputIcon || <CalendarIcon size={iconSize} />}
-        margin={reverse ? { left: 'small' } : { right: 'small' }}
+        margin={
+          reverse
```

Comment:
Just thinking about how we'll document -- it seems like typically the value is the same and it's just the side that changes based on if `reverse` is used.

Do we want to have 2 different theme properties "left" and "right"? Or just a single one to start, then could always extend to support an object in future.

Thinking about it from the consumer perspective, I feel like it's unlikely that we'd be defining something different for when it's reverse vs not.

Reply:
Although, soft opinion.

Reply:
Changed this in latest commit